TestApp/app/App.js

```diff
@@ -13,24 +13,27 @@ import AnalyticsScreen from './screens/AnalyticsScreen';
 import CrashesScreen from './screens/CrashesScreen';
 import AttachmentsProvider from './AttachmentsProvider';
 
-// LogBox.ignoreWarnings(['Remote debugger']);
+const Tab = createBottomTabNavigator();
 
-const TabNavigator = createBottomTabNavigator(
-  {
-    screens: {
-      AppCenter: AppCenterScreen,
-      Analytics: AnalyticsScreen,
-      Transmission: TransmissionScreen,
-      Crashes: CrashesScreen
-    },
-    tabBarOptions: {
-      activeBackgroundColor: 'white',
-      inactiveBackgroundColor: 'white',
-    },
-  }
-);
+const App = () => {
+  return (
+    <NavigationContainer>
+      <Tab.Navigator
+        tabBarOptions={{
+          activeBackgroundColor: 'white',
+          inactiveBackgroundColor: 'white',
+        }}
+      >
+        <Tab.Screen name="AppCenter" component={AppCenterScreen} />
```

Comment:
Seems like tab icons aren't loaded for some reason, can you check it out?
<img width="349" alt="Screenshot 2024-11-22 at 11 29 35" src="https://github.com/user-attachments/assets/0d12a449-49e6-4df1-bf6f-36f5d7ffa72e">

Reply:
fixed